### PR TITLE
Change title color to purple

### PR DESCRIPTION
**Pull request by varsha-bhar**
i like purple.

before:
image

after:
image

### CODE REVIEW

**Comment by lhlRahman** [comment]
kinda bad lol

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
             <button id="tab-quiz" class="tablink" onclick="handleTabSelection('tab-quiz')">Quiz</button>
         </div>
         <div id="fundamentals" class="tabcontent">
-            <h2 id="title1">Git Fundamentals</h2>
+            <h2 id="title1">Git Fundamentals big</h2>
```
Comment (SaintofHeaven) [inline index.html]: Very poor reasoning  for this change smh

File: styles.css [hunk 2]
```diff
@@ -4,6 +4,7 @@ body {
 
 #title1 {
     color: blue;
+    font-size: 96px;
```
Comment (banyar-shin) [inline styles.css]: We should not overwrite the size of h2 tags.

File: index.html (hunk 1 above)
Comment (banyar-shin) [inline index.html]: There is no reason to add the word big.

**Review by banyar-shin — CHANGES_REQUESTED** [review]
Please make some changes ASAP. See comments.

**Comment by jzhangg917** [comment]
What were you thinking?????????

File: styles.css (hunk 2 above)
Comment (imrenmore) [inline styles.css:7]: Maybe need new prescription

Comment (varsha-bhar) [reply styles.css]: piss off.

Comment (varsha-bhar) [inline styles.css]: i got a new one!

**Review by banyar-shin — APPROVED** [review]
Looks good to me but please update your PR title and description!

**Pull request merged**